addons/mass_mailing/models/mailing.py

```diff
@@ -540,28 +545,48 @@ def action_view_delivered(self):
         return self._action_view_documents_filtered('delivered')
 
     def _action_view_documents_filtered(self, view_filter):
-        if view_filter in ('reply', 'bounce'):
+        model_name = self.env['ir.model']._get(self.mailing_model_real).display_name
+        helper = None
+        if view_filter == 'reply':
+            found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status == view_filter)
+            helper = (
+                _("No %s replied to your mailing yet!", model_name),
+                _("To track how many replies this mailing gets, make sure its reply-to address belongs to this database.")
+            )
+        elif view_filter == 'bounce':
             found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status == view_filter)
+            helper = (
+                _("No %s address bounced yet!", model_name),
+                _("Bounce happens when a mailing cannot be delivered (fake address, server issues, ...). Check each record to see what went wrong.")
+            )
         elif view_filter == 'open':
             found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status in ('open', 'reply'))
-        elif view_filter == 'click':
-            found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.links_click_datetime)
+            helper = (
+                _("No %s opened your mailing yet!", model_name),
+                _("Come back once your mailing has been sent to track who opened your mailing.")
+            )
         elif view_filter == 'delivered':
             found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status in ('sent', 'open', 'reply'))
+            helper = (
+                _("No %s received your mailing yet!", model_name),
+                _("Wait until your mailing has been sent to check how many recipients you managed to reach.")
+            )
         elif view_filter == 'sent':
             found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.sent_datetime)
         else:
             found_traces = self.env['mailing.trace']
         res_ids = found_traces.mapped('res_id')
-        model_name = self.env['ir.model']._get(self.mailing_model_real).display_name
-        return {
+        action = {
             'name': model_name,
             'type': 'ir.actions.act_window',
             'view_mode': 'tree',
             'res_model': self.mailing_model_real,
             'domain': [('id', 'in', res_ids)],
             'context': dict(self._context, create=False)
         }
+        if helper:
+            action['help'] = '<p class="o_view_nocontent_smiling_face">' + helper[0] + '</p><p>' + helper[1] + '</p>'
```

Comment:
Same global remark: would define helper_header / helper_message and use string formatting here, just for code clarity